internal/storage/vmbus/vmbus.go

```diff
@@ -0,0 +1,43 @@
+// +build linux
+
+package vmbus
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+var Filepathglob = filepath.Glob
```

Comment:
Exporting this so we can replace it for testing seems a bit gross to me. If we can find a cleaner way to do it, that would be nice. If not, let's at least clearly document here why this is exposed this way.

Reply:
Found the comment with Justin where he recommended doing it this way. :) I think that's fine, testing is always grosser than I want it to be. Please just add a comment here explaining.

Reply:
Done!